Docs/install/index.rst

```diff
@@ -34,118 +34,175 @@
+
+With CUDA 11.x:
 
 .. parsed-literal::
 
-    # ONNX 1.16 GPU with CUDA 11.x
-    python3 -m pip install |download_url|\ |version|/aimet_onnx-\ |version|.cu117\ |whl_suffix| -f |torch_pkg_url|
+    python3 -m pip install |download_url|\ |version|/aimet_onnx-\ |version|.cu117\ |whl_suffix|
 
-    # ONNX 1.16 CPU
-    python3 -m pip install |download_url|\ |version|/aimet_onnx-\ |version|.cpu\ |whl_suffix| -f |torch_pkg_url|
+With CPU only:
 
+.. parsed-literal::
 
-For older versions, please browse the releases at https://github.com/quic/aimet/releases and follow the documentation corresponding to that release to select and install the appropriate package.
+    python3 -m pip install |download_url|\ |version|/aimet_onnx-\ |version|.cpu\ |whl_suffix|
 
-.. |whl_suffix| replace:: -cp310-cp310-manylinux_2_34_x86_64.whl
-.. |download_url| replace:: \https://github.com/quic/aimet/releases/download/
-.. |torch_pkg_url| replace:: \https://download.pytorch.org/whl/torch_stable.html
 
-System Requirements
-~~~~~~~~~~~~~~~~~~~
+Next steps
+----------
 
-The AIMET package requires the following host platform setup:
+See the :doc:`Quantization User Guide </user_guide/model_quantization>` for a discussion of how to use AIMET quantization.
+
+See the :doc:`Examples Documentation </user_guide/examples>` to try AIMET on example quantization and compression problems.
 
-* 64-bit Intel x86-compatible processor
-* Linux Ubuntu: 22.04 LTS
-* bash command shell
-* For GPU variants:
-    * Nvidia GPU card (Compute capability 5.2 or later)
-    * nvidia-docker - Installation instructions: https://github.com/NVIDIA/nvidia-docker
 
-To use the GPU accelerated training modules an Nvidia CUDA enabled GPU with a minimum Nvidia driver version of 455+ is required. Using the latest driver is always recommended, especially if using a newer GPU. Both CUDA and cuDNN (the more advanced CUDA interface) enabled GPUs are supported.
+.. _idx-install-older:
 
+Installing an older version
+===========================
 
-Advanced Installation Instructions
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+View the release notes for older versions at https://github.com/quic/aimet/releases. Follow the documentation corresponding to that release to select and install the appropriate package.
+
+.. |whl_suffix| replace:: -cp310-cp310-manylinux_2_34_x86_64.whl
+.. |download_url| replace:: \https://github.com/quic/aimet/releases/download/
 
-There are two ways to setup and install AIMET:
-    * On your host machine
-    * Using our pre-built development `Docker images <https://artifacts.codelinaro.org/ui/native/codelinaro-aimet/aimet-dev>`_
 
-Please click on the appropriate link for installation instructions:
+:hideitem:`Other installation options`
+======================================
 
 .. toctree::
    :titlesonly:
-   :maxdepth: 3
+   :hidden:
+   :maxdepth: 2
 
-   Install in Host Machine <install_host>
-   Install in Docker Container <install_docker>
+   Manual install <install_host>

```

Comment:
I feel the previous titles were more accurate. Both are effectively "manual install", one offers a docker container that helps skip some steps.

Reply:
Changed back.